src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -2,6 +2,23 @@ import React from 'react';
 import { Card } from 'antd';
 import { dateConverter } from '../../common/dateHelpers';
 import { timeConverter } from '../../common/timeHelpers';
+import { Button } from '../../common';
+import axios from 'axios';
+// [POST]   | /course/ | Contains fields: `description`, `subject`, and  `prereq`. Returns the event object with the specified `subject`.
+// const handleClick = () => {
+
+//   axios.post('/course/', {
+//     description: 'Fred',
+//     subject: 'Flintstone'
+
+//   })
+//   .then(function (response) {
+//     console.log(response);
+//   })
+//   .catch(function (error) {
+//     console.log(error);
+//   });
+// }
```

Comment:
Does this commented-out code need to be merged into our main codebase? Maybe there's a decision to make about whether you want to finish implementing whatever was started here or get rid of it. We also want to consider the advantages and disadvantages of making an API call locally vs adding it to our global store and pulling from there.

Reply:
Great catch! Since Elchin and I are working on the same task, I placed the commented code where we might set up the api post to add a course to the students "My Courses". I was also going back and forth on our branch and on my stash.